Contained/Contained/Files/SettingsViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SettingsViewController.swift
+//  Contained
+//
+//  Created by Dojo on 6/3/20.
+//  Copyright © 2020 Dojo. All rights reserved.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+
+    @IBAction func toggleZoom(_ sender: UISwitch) {
+        Settings.shared.shouldRoll = sender.isOn
+    }
+    
+    @IBAction func toggleRoll(_ sender: UISwitch) {
+        Settings.shared.shouldZoom = sender.isOn
+    }
```

Comment:
These are swapped (oops) and not actually hooked up to the storyboard. Fixing that, it works great!